pkgs/by-name/heads/package.nix

```
@@ -0,0 +1,302 @@
+{
+  stdenv,
+  lib,
+  fetchgit,
+  fetchFromGitHub,
+  fetchurl,
+  replaceVars,
+  runCommandNoCC,
+  unstableGitUpdater,
+  writeShellApplication,
+  _experimental-update-script-combinators,
+  bashNonInteractive,
+  bc,
+  bison,
+  ccache,
+  cmake,
+  common-updater-scripts,
+  cpio,
+  curl,
+  elfutils,
+  envsubst,
+  flex,
+  getopt,
+  gnat,
+  gnat-bootstrap,
+  gnum4,
+  gnumake,
+  git,
+  imagemagick,
+  jq,
+  ncurses,
+  newScope,
+  nix,
+  nixfmt-rfc-style,
+  nix-prefetch-scripts,
+  perl,
+  pkg-config,
+  rsync,
+  xz,
+  zip,
+  zlib,
+  brandName ? "Heads",
```

It would probably be nice to sort the deps for better readability

```suggestion
  lib,
  stdenv,

  fetchgit,
  fetchFromGitHub,
  fetchurl,

  replaceVars,
  runCommandNoCC,
  newScope,

  bashNonInteractive,

  # build deps
  bc,
  bison,
  ccache,
  cmake,
  cpio,
  curl,
  flex,
  gnum4,
  git,
  imagemagick,
  ncurses,
  perl,
  pkg-config,
  rsync,
  zip,
  gnat,
  gnat-bootstrap,

  # runtime deps
  elfutils,
  zlib,

  # update script
  unstableGitUpdater,
  writeShellApplication,
  _experimental-update-script-combinators,
  envsubst,
  getopt,
  gnumake,
  jq,
  nix,
  nixfmt-rfc-style,
  nix-prefetch-scripts,
  xz,

  brandName ? "Heads",
```

---

Not a fan of these group labels (i.e. `# update script`) - dependencies may over time move between them, or belong into multiple at the same time. But I will try to resort them into roughly:
```nix
{
  lib,
  stdenv,
  # Nix functions
  # Packages used to build time
  # Variables
}:
```

---

I know that this might be a matter of preference so this is merely a suggestion, but even having them separated by spaces is more readable to me and easier to discern which parts are related, as opposed to having a long list of everything.

---

Spaces added.